test/functional/tool_signet_miner.py

```diff
@@ -12,14 +12,26 @@
 
 from test_framework.blocktools import DIFF_1_N_BITS
 from test_framework.key import ECKey
-from test_framework.script_util import key_to_p2wpkh_script
+from test_framework.script_util import CScript, key_to_p2wpkh_script
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal
 from test_framework.wallet_util import bytes_to_wif
 
 
 CHALLENGE_PRIVATE_KEY = (42).to_bytes(32, 'big')
+SIGNET_COMMITMENT = 'ecc7daa2'
```

Comment:
for consistency, would use the same name here as in the C++ codebase, i.e. `SIGNET_HEADER`
https://github.com/bitcoin/bitcoin/blob/3edf400b1020d7b88402ebc0e758b1fad2e7a781/src/signet.cpp#L27
could also move this to a test_framework module (maye `blocktools.py`) so it can be reused in the signet miner script